apps/remix-ide/src/app/ui/multiParamManager.js

```diff
@@ -6,6 +6,25 @@ var copyToClipboard = require('./copy-to-clipboard')
 var remixLib = require('@remix-project/remix-lib')
 var txFormat = remixLib.execution.txFormat
 
+function stringToBytes32(str) {
+  const bytes = new TextEncoder().encode(str)
+  if(bytes.length > 32) {
+    // TODO find a better way to report the error
```

Comment:
Please add an issue and mention the number here if you can't avoid todos
@todo(#number) - description